tests/TestCase/Core/ConfigureTest.php

```diff
@@ -69,6 +69,35 @@ public function tearDown()
     }
 
     /**
+     * testReadOrFail method
+     *
+     * @return void
+     */
+    public function testReadOrFail()
+    {
+        $expected = 'ok';
+        Configure::write('This.Key.Exists', $expected);
+        $result = Configure::readOrFail('This.Key.Exists');
+        $this->assertEquals($expected, $result);
+    }
+
+    /**
+     * testReadOrFail method
+     *
+     * @return void
+     */
+    public function testReadOrFailThrowingException()
+    {
+        try {
+            $void = Configure::readOrFail('This.Key.Does.Not.exist');
+            $this->fail('Expected exception to be thrown.');
+        } catch (\Exception $e) {
+            $this->assertTrue($e instanceof \RuntimeException);
+            $this->assertEquals('Expected "This.Key.Does.Not.exist" configuration.', $e->getMessage());
```

Comment:
* So is there some need to change this?
* Is it enough to add that docblock comment to achieve what I did with try/catch?
* The try/catch pattern is used below I think? It is also very transparent what's happening